config/params.php

```diff
@@ -55,31 +55,25 @@
         ],
     ],
 
-    'yiisoft/forms' => [
-        'field' => [
-            'ariaDescribedBy' => [true],
-            'containerClass' => ['form-floating mb-3'],
-            'errorClass' => ['fw-bold fst-italic invalid-feedback'],
-            'hintClass' => ['form-text'],
-            'inputClass' => ['form-control'],
-            'invalidClass' => ['is-invalid'],
-            'labelClass' => ['floatingInput'],
-            'template' => ['{input}{label}{hint}{error}'],
-            'validClass' => ['is-valid'],
-            'defaultValues' => [
-                [
-                    'submit' => [
-                        'definitions' => [
-                            'class()' => ['btn btn-primary btn-lg mt-3'],
-                        ],
-                        'containerClass' => 'd-grid gap-2 form-floating',
+    'yiisoft/form' => [
+        'configs' => [
+            'default' => [
```

Comment:
What's the benefit configuring form defaults instead of widgets?

Reply:
> What's the benefit configuring form defaults instead of widgets?

This configuration of `FieldFactory`, that used for configure all fields widgets. For configure widgets via `yiisoft/widgets` we would have to set this paramateres for each widgets and for each new or custom widgets we also would also have to add this parameters.